conductr_cli/test/conduct_load_test_base.py

```diff
@@ -64,23 +63,20 @@ def base_test_success(self):
         with patch('conductr_cli.resolver.resolve_bundle', resolve_bundle_mock), \
                 patch('requests.post', http_method), \
                 patch('builtins.open', open_mock), \
-                patch('conductr_cli.conduct_url.request_headers', request_headers_mock), \
                 patch('conductr_cli.bundle_installation.wait_for_installation', wait_for_installation_mock):
             logging_setup.configure_logging(input_args, stdout)
             result = conduct_load.load(input_args)
             self.assertTrue(result)
 
         open_mock.assert_called_with(self.bundle_file, 'rb')
         resolve_bundle_mock.assert_called_with(self.custom_settings, self.bundle_resolve_cache_dir, self.bundle_file)
-        request_headers_mock.assert_called_with(input_args)
         http_method.assert_called_with(self.default_url, files=self.default_files, timeout=LOAD_HTTP_TIMEOUT,
-                                       headers=self.mock_headers)
+                                       headers={'Host': '127.0.0.1'})
```

Comment:
We were previously checking the host header being set up via another mock. The other mock was no longer available due to its function having been replaced. The code should also be much simpler now.